src/routes/dev/menu/simple-selector-menu.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+  import { SelectMenu } from "$lib/components/menu";
+  import Tooltip from "$lib/components/tooltip/Tooltip.svelte";
+  import TooltipContent from "$lib/components/tooltip/TooltipContent.svelte";
+
+  const singleSelectorOptions = [
+    { key: 0, main: "option 1", right: "opt1" },
+    { key: 1, main: "option 2", right: "opt2" },
+    { key: 2, main: "option 3", right: "opt3" },
+    {
+      key: 3,
+      main: "option 4",
+      right: "opt4",
+      description: "triggers error state",
+    },
+    { key: 4, main: "option 5", right: "opt5" },
+  ];
+  let singleSelections = [singleSelectorOptions[0]];
+
+  let level = undefined;
+  $: if (singleSelections[0]?.description) {
+    level = "error";
+  } else {
+    level = undefined;
+  }
+</script>
+
+<section class="grid grid-flow-row gap-y-4">
+  <h1 class="text-lg">Select Menus</h1>
+
+  <p>
+    the <code>SelectMenu</code> component takes care of the basic cases covered
+    by the {`<select>`}
+    element. It has a few additional bells and whistles:
+  </p>
+
+  <ul>
+    <li>you can add descriptions & right-text to the menu items</li>
+    <li>you can change the styling of the overall element</li>
+    <li>it's easy to make a block-level select element</li>
+    <li>
+      tooltips attached to the top-level element will suppress when active
+    </li>
+    <li>
+      you can add a level prop, which currently accepts "error" and turns the
+      element red
+    </li>
+  </ul>
+
+  <h2 class="text-md">basic inline select</h2>
+
+  <div class="grid grid-flow-row gap-y-4" style:width="600px">
+    <div>
+      Currently selecting
+      <Tooltip distance={16}>
+        <SelectMenu
+          {level}
+          options={singleSelectorOptions}
+          bind:selections={singleSelections}
+        />
+        <TooltipContent slot="tooltip-content">
+          {#if level}
+            {level}
+          {:else}
+            a simple selector. This will disappear on click
+          {/if}
+        </TooltipContent>
+      </Tooltip>
+    </div>
+
+    <div class="pl-4">
+      {#each singleSelections as option}
```

Comment:
Since this is just single-select, shouldn't I expect the bind variable to always point to a single object instead of an array of objects?

Reply:
You're right, I shoudl change this back. Originally I had htis working with muli-select as well but didn't really want to go too deep on this yet until we think through what we want there.